packages/maskbook/src/utils/hooks/useSuspense.ts

```diff
@@ -0,0 +1,27 @@
+import { useUpdate } from 'react-use'
+
+export function useSuspense<T, U extends Array<any>>(
```

Comment:
Please make sure your implementation follow the contract described in the https://github.com/facebook/react/issues/17526#issuecomment-769151686